domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/PartAdmissionResponse.java

```diff
@@ -0,0 +1,158 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
+import uk.gov.hmcts.cmc.domain.models.party.Party;
+import uk.gov.hmcts.cmc.domain.models.response.DefendantPaymentPlan;
+import uk.gov.hmcts.cmc.domain.models.response.EvidenceItem;
+import uk.gov.hmcts.cmc.domain.models.response.HowMuchOwed;
+import uk.gov.hmcts.cmc.domain.models.response.HowMuchPaid;
+import uk.gov.hmcts.cmc.domain.models.response.PayBySetDate;
+import uk.gov.hmcts.cmc.domain.models.response.TimelineEvent;
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+public class PartAdmissionResponse extends Response {
+
+    public enum PartAdmissionType {
+        AMOUNT_TOO_HIGH,
+        PAID_WHAT_BELIEVED_WAS_OWED
+    }
+
+    @NotNull
+    private final PartAdmissionType partAdmissionType;
+
+    @Valid
+    @NotNull
+    private final ImmutableList<EvidenceItem> evidence;
+
+    @Valid
+    @NotNull
+    private final ImmutableList<TimelineEvent> timeline;
+
+    @Valid
+    @NotNull
+    private final HowMuchOwed howMuchOwed;
```

Comment:
I think we cannot have both `howMuchOwed` and `howMuchPaid` at the same time? Did you manage to submit response locally?

Reply:
Yes I think you maybe right, I haven't tested response submit locally

Reply:
Please confirm it, as I know the service these two options are mutually exclusive.